DTMD.xsd

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:vc="http://www.w3.org/2007/XMLSchema-versioning" xmlns:dtmd="http://apps.pmsf.net/STMD/DecisionTaskMetaData" xmlns:stc="http://apps.pmsf.net/SSPTraceability/SSPTraceabilityCommon" xmlns:ssc="http://ssp-standard.org/SSP1/SystemStructureCommon" targetNamespace="http://apps.pmsf.net/STMD/DecisionTaskMetaData" elementFormDefault="qualified" vc:minVersion="1.1">
```

Comment:
Is xmlns:dtmd="http://apps.pmsf.net/**STMD**/DecisionTaskMetaData correct?

Reply:
Jain, historisch war alles in STMD, also STMD, SRMD, und nun auch DTMD; als SSC eingeführt wurde hatte ich aber auf SSPTraceability gewechselt, weil das da dann schon klarer war. Im Endeffekt ist es egal, da ja sowieso dann auf ssp-standard.org URIs gewechselt wird... Kann ich auf SSPTraceability anpassen, die Frage ist, führt das bei euch zu Problemen?

Reply:
No, that's no problem for us.